v-next/hardhat-ignition-ethers/test/test-helpers/create-hre.ts

```diff
@@ -0,0 +1,39 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types/hre";
+import type { NetworkConnection } from "hardhat/types/network";
+
+import path from "node:path";
+import { pathToFileURL } from "node:url";
+
+import { createHardhatRuntimeEnvironment } from "hardhat/hre";
+
+import hardhatIgnitionEthers from "../../src/index.js";
+
+export async function createConnection(): Promise<NetworkConnection> {
+  const hre = await createHRE();
+  const connection = await hre.network.connect();
+
+  await connection.provider.request({
+    method: "evm_setAutomine",
+    params: [true],
+  });
+
+  return connection;
+}
+
+export async function createHRE(): Promise<HardhatRuntimeEnvironment> {
```

Comment:
This function doesn’t need to be exported.

Nit: we usually use camelCase, even for acronyms.

(Same comment for `v-next/hardhat-ignition-viem/test/test-helpers/create-hre.ts`)

Reply:
Ah good catch. 

In regards to the camel case, I was following the pattern of [this hardhat test](https://github.com/NomicFoundation/hardhat/blob/v-next/v-next/hardhat/test/internal/builtin-plugins/flatten/task-action.ts#L52), but I can change it if needed.

Reply:
Yeah, I think this is a mistake (probably mine 😅).